src/components/StrategyTradesTable/StrategyTradesTable.helpers.js

```diff
@@ -73,7 +73,7 @@ const onTradeExportClick = (rawTrades, results, activeMarket, t) => {
 
   csvExport.export(documents, (buffer) => {
     const blob = new Blob([buffer], { type: 'application/zip' })
-    const filename = getExportFilename(activeMarket)
+    const filename = getExportFilename(`${activeMarket.base}-${activeMarket.quote}`)
```

Comment:
lets use getPairFromMarket helper so name would be similar to what they see in markets dropdown

`export const getPairFromMarket = (market, getCurrencySymbol) => (market?.isPerp ? market.uiID : `${getCurrencySymbol(market?.base)}/${getCurrencySymbol(market?.quote)}`)`

Reply:
I'm using this format because on linux filenames cannot contain `/`..